### PR TITLE
Time picker custom label

### PR DESCRIPTION
**Pull request by valadzhov**
Closes # 

### Checklist:
 
 - [ ] check topic's TOC/menu and paragraph headings
 - [ ] link to other topics using `../relative/path.md`
 - [ ] at the References section at the end of the topic add links to topics, samples, etc
 - [ ] reference API documentation instead of adding a section with API

<br />

 - [ ] use valid component names - [Data] Grid, `IgxSelectComponent`, `<igx-combo>`
 - [ ] use spell checker tool (VS Code, Grammarly, Microsoft Editor)
 - [ ] add inline `code blocks` for the names of classes / tags / properties
 - [ ] add language descriptor for the ```code blocks```
 - [ ] check broken links (use browser add-on)
 - [ ] check if sample is working and fully visible in the topic
 - [ ] check if sample is working and fully visible in the StackBlitz
 - [ ] check if code blocks match the code in StackBlitz demo

<br />

 - [ ] follow SEO guidelines to fill topic's metadata ([SEO guidelines](https://infragisticsinc297.sharepoint.com/:w:/g/Groups/marketing/EWz9InT4FDlErHCumxsKGY4Bd8H03yhRWxDFk47luRz-_Q?e=S5wWcx))

<br />

 - [ ] do not resolve requested changes (leave that to the reviewer)
 - [ ] add `pending-localization` label when the review of the PR is done
 - [ ] add a member from the localization team to translate it

### CODE REVIEW

File: en/components/time_picker.md [hunk 1]
```diff
@@ -331,6 +331,26 @@ And there we have it, a re-templated time picker with dropdown, custom actions a
 </div>
 <div class="divider--half"></div>
 
+#### Custom label
+By placing a `<label>` inside the tag of the component the default label can be changed to a custom one:
```
Comment (damyanpetev) [inline en/components/time_picker.md:335]: marked with `igxLabel` directive

Comment (valadzhov) [reply en/components/time_picker.md:335]: The sentence was changed to "By placing an `igxlabel` inside a `<label>` tag nested in the component, the default label can be changed to a custom one:".

File: en/components/time_picker.md [hunk 2]
```diff
@@ -90,6 +90,19 @@ The table below lists valid time display formats:
 | `mm` | Formats the minutes field with leading zero (00..59). |
 | `tt` | Represents the AM/PM field. |
 
+#### Custom label
+By placing an `igxlabel` inside a `<label>` tag nested in the component, the default label can be changed to a custom one:
```
Comment (damyanpetev) [inline en/components/time_picker.md:94]: `igxlabel` directive on a `<label>` tag

File: en/components/time_picker.md [hunk 3]
```diff
@@ -90,6 +90,19 @@ The table below lists valid time display formats:
...
+
+````html
+<!-- timepicker-label.component.html -->
+<igx-time-picker>
+    <label igxLabel>Meeting Start</label>
+</igx-time-picker>
+````
+
+You can see what the custom label will look like in the validation sample found in the Validation section.
```
Comment (damyanpetev) [inline en/components/time_picker.md:103]: maybe with a link?
```suggestion
You can see what the custom label will look like in the validation sample found in the [Validation section](#validation).
```

Comment (valadzhov) [reply en/components/time_picker.md]: It's done!

**Review by Lipata — APPROVED** [review]

**Pull request merged**